pages.de/common/terraform-plan.md

```diff
@@ -0,0 +1,32 @@
+# terraform plan
+
+> Erzeugen und Anzeigen von Terraform-Ausführungsplänen.
+> Mehr Informationen: <https://www.terraform.io/docs/cli/commands/plan.html>.
+
+- Erzeugen und Anzeigen des Ausführungsplans im aktuellen Verzeichnis:
+
+`terraform plan`
+
+- Einen Plan zur Zerstörung aller derzeit existierenden entfernten Objekte anzeigen:
+
+`terraform plan -destroy`
+
+- Anzeigen eines Plans zur Aktualisierung des Terraform-Status und der Ausgabewerte:
+
+`terraform plan -refresh-only`
+
+- Werte für Eingabevariablen festlegen:
+
+`terraform plan -var '{{name1}}={{value1}}' -var '{{name2}}={{value2}}'`
+
+- Anzeigen eines Plans auf eine Teilmenge von Ressourcen:
+
+`terraform plan -target {{resource_type.resource_name[instance index]}}`
```

Comment:
Can you translate the command argument `{{resource_type.resource_name[instance index]}}`?